cli/valet.php

```diff
@@ -495,7 +495,18 @@
      */
     $app->command('use [phpVersion] [--force]', function ($phpVersion, $force) {
         if (!$phpVersion) {
-            return info('Valet is using ' . Brew::linkedPhp());
+            $path = getcwd() . '/.valetrc';
+            $linkedVersion = Brew::linkedPhp();
+            if (!file_exists($path)) {
+                return info(sprintf('Valet is using %s.', $linkedVersion));
+            }
+
+            $phpVersion = trim(file_get_contents($path));
+            info('Found \'' . $path . '\' with version: ' . $phpVersion);
```

Comment:
Instead of `with version:` perhaps `requesting version:` or `specifying version:` might be more accurate?

Reply:
Yes this makes sense, I'll update this PR shortly with these changes